src/state/slices/common-selectors.ts

```diff
@@ -200,6 +200,24 @@ export const selectAssetsSortedByMarketCapUserCurrencyBalanceAndName =
     },
   )
 
+export const selectAssetsSortedByMarketCapAndName = createDeepEqualOutputSelector(
+  selectAssets,
+  selectMarketDataUsd,
+  (assets, marketDataUsd) => {
+    // This looks weird but isn't - looks like we could use the sorted selectAssetsByMarketCap instead of selectAssets
+    // but we actually can't - this would rug the triple-sorting
```

Comment:
```suggestion
    // but we actually can't - this would rug the double-sorting
```

Reply:
Changed to quadruple in [a9e1e9b](https://github.com/shapeshift/web/pull/8175/commits/a9e1e9be6891e1fa9ed75b55dbcb3364493ea0d9) as it changed a little bit